CHANGES.txt

```diff
@@ -1,119 +1,116 @@
-CHANGES
--------
-
 0.16.0 (2019-01-25)
 ^^^^^^^^^^^^^^^^^^^
 
-* Fix select priority name (#525)
+* Fix select priority name `#525 <https://github.com/aio-libs/aiopg/issues/525>`_
```

Comment:
Maybe add extlinks for this?
https://github.com/cherrypy/cheroot/blob/92490dd/docs/conf.py#L34-L40